src/runtime/ui.config/elements/feed.ts

```diff
@@ -0,0 +1,52 @@
+export default {
+  base: 'grid gap-x-4',
+  style: {
+    'grid-template-columns': 'max-content max-content max-content',
```

Comment:
Maybe `'max-content minmax(0, 1fr) max-content'` is better? The second column may be very wide because of long description. We could make it flexible to avoid overflow.

Reply:
I think I remember why I used this, and was because if you do it, the other could get squashed making double lines unnecessarily.